### DISCUSSION TITLE
Add setting to not keep note title up-to-date with event title

### DISCUSSION THREAD

**Discussion by HEmile** [davish/obsidian-full-calendar#357] [Feature Requests]
This is a personal gripe, but I don't like including metadata into my notes titles. However, when upgrading an event to Full Calendars format (or when creating one), it automatically adds the date of the note to the title. 

I tried renaming the notes to remove the date, and it seems like Full Calendar still picks those up, so I'm pretty sure there's no technical reason for this.

**Comment by HEmile** [comment]
Another even more important reason this shouldn't be the default is that the backlinks to the event's note break since those are not adapted to include the date. (To be fair this can be considered a bug report on its own!)

**Comment by davish** [comment]
@HEmile thanks for your comment! I added a separate bug report for that. It's not strictly necessary for the date to be in the event title, but the issue is that if one is to keep the event title in sync with the note title, then there needs to be some way to distinguish between two events with the same name. I understand not everyone wants this use case, so I'll change the title of this issue to reflect that adding a setting for this would stop note titles from being kept up-to-date with event titles after creation.
